### DISCUSSION TITLE
Modernize navbar design

### DISCUSSION THREAD

**Discussion by inwardmovement** [twbs/bootstrap#36448] [General]
What do you think about improving the navbar like so?

- Navbars are often sticky, which goes not so well with the collapse behavior. What about using a kind of dropdown/offcanvas by default instead?

- The expand button would have a better UX if it was moved outside the navbar, like a floating button at the bottom of the (mobile) screen.

**Comment by mdo** [comment]
We have the offcanvas option now, which I could see us switching to completely (or as the default) in v6. Anyone can put an expand button outside the navbar, but perhaps we can add an example for that.
